README.md

```diff
@@ -4,35 +4,44 @@ OpenTelemetry aware logger for Cloudflare Workers and Vercel Edge Functions.
 
 ## Usage
 
-```
+```bash
 npm i @baselime/edge-logger
 ```
 
+Add the API key
+
+```bash
+npx wrangler secret put BASELIME_KEY <your Baselime api key>
+```
+
 Add the Logger to your Cloudflare worker
 
-```javascript
+```typescript
+import { BaselimeLogger } from '@baselime/edge-logger'
+
+export interface Env {
+	BASELIME_KEY: string
+}
+
 export default {
-    async fetch(req: Request, env: Env, ctx: ExecutionContext): Promise<Response> {
+	async fetch(req: Request, env: Env, ctx: ExecutionContext): Promise<Response> {
 		const logger = new BaselimeLogger({
 			ctx,
 			apiKey: env.BASELIME_KEY,
 			service: 'my-worker',
 			dataset: 'cloudflare',
 			namespace: 'fetch',
-			requestId: req.headers.get('cf-ray'),
+			requestId: crypto.randomUUID(),
```

Comment:
cf-ray is not guaranteed to be unique between requests: https://developers.cloudflare.com/fundamentals/reference/cloudflare-ray-id/

Reply:
TIL, thank you!